utils/_context/_scenarios/__init__.py

```diff
@@ -601,13 +608,21 @@ class _Scenarios:
             "DD_TRACE_PROPAGATION_BEHAVIOR_EXTRACT": "restart",
             "DD_TRACE_PROPAGATION_EXTRACT_FIRST": "true",
             "DD_LOGS_INJECTION": "true",
-            "DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED": "false",
         },
         appsec_enabled=False,
         doc="",
         scenario_groups=[ScenarioGroup.TRACING_CONFIG],
     )
 
+    tracing_config_nondefault_4 = EndToEndScenario(
```

Comment:
Can you add this scenario in `.github/workflows/run-end-to-end.yml` ?

Reply:
Yep, done in ed6c49a2bff8aae42f421dd4b5cd1d784cbb57b1